src/uncrustify.cpp

```diff
@@ -181,10 +226,11 @@ static void usage_exit(const char *msg, const char *argv0, int code)
            "                            Detection is fairly limited.\n"
            "\n"
            "Debug Options:\n"
-           " -p FILE      : dump debug info to a file\n"
-           " -L SEV       : Set the log severity (see log_levels.h)\n"
-           " -s           : Show the log severity in the logs\n"
-           " --decode     : decode remaining args (chunk flags) and exit\n"
+           " -p FILE          : dump debug info to a file\n"
+           " -L SEV1 SEV2 ... : Use 'a' for all levels. See the log severity availibles on\n"
```

Comment:
That isn't how the logsev works.
It is a comma-separated list of numbers or ranges of numbers.
Example:  -L0-3,45,56-58